### DISCUSSION TITLE
A way to configure the trigger button threshold before the ZL/ZR buttons register a press?

### DISCUSSION THREAD

**Discussion by 20excal07** [ndeadly/MissionControl#637] [Ideas]
For the controllers that have analog triggers. Personally I have a Dualsense, and I find that I tend to lightly fumble with the triggers when my hands are idling, but since it registers a press at even the slightest push of the triggers, it gets a bit annoying when the buttons actually do something (e.g. during a cutscene where pressing them would skip it). It'd be nice if there's a setting to configure how much the trigger has to be pushed in before it registers a ZL/ZR press. (even just having them register at 20% would be super nice)

**Comment by ndeadly** [comment]
Sure, it could be added. Open this as a feature request on the issue tracker.

**Reply by 20excal07** [reply]
done!
